test/e2e/parallel/user_workloads_test.go

```diff
@@ -231,6 +247,39 @@ func createAAP(t *testing.T, memberAwait *wait.MemberAwaitility, name, namespace
 	return deployment
 }
 
+// createNotebook creates an instance of notebooks.kubeflow.org with one StatefulSet owned by this instance
+// returns the underlying StatefulSet
+func createNotebook(t *testing.T, memberAwait *wait.MemberAwaitility, name, namespace string) *appsv1.StatefulSet {
+	clnt, err := dynamic.NewForConfig(memberAwait.RestConfig)
+	require.NoError(t, err)
+
+	// Create a Notebook instance
+	notebook := notebookResource(name)
+	createdNotebook, err := clnt.Resource(notebookRes).Namespace(namespace).Create(context.TODO(), notebook, metav1.CreateOptions{})
+	require.NoError(t, err)
+
+	// Create a StatefulSet with one replica and the Notebook instance as the owner
+	replicas := int32(1)
+	statefulSet := &appsv1.StatefulSet{
```

Comment:
Just curious, why StatefulSet instead of using a Deployment? Is it because the actual underlying workload for the notebook is a StatefulSet or are there other reasons?